internal/workflow.go

```diff
@@ -380,9 +380,18 @@ type (
 		// WARNING: Worker versioning is currently experimental
 		VersioningIntent VersioningIntent
 
-		// TODO(cretz): Expose once https://github.com/temporalio/temporal/issues/6412 is fixed
```

Comment:
I don't think we need to change anything related to static summary/details or child workflows in this timer-specific issue. Also, while that issue is technically fixed, it would need to be in the dev server and we need integration tests.

Reply:
sounds good, I'll take these changes out of this PR